docs/guides/insar_product_guide.md

```diff
@@ -191,11 +191,15 @@ All of the phase differences in a wrapped interferograms lie between -π and π.
 Before the interferogram can be unwrapped, it must be filtered to remove noise. This is accomplished using an adaptive spectral filtering algorithm. This adaptive interferogram filtering aims to reduce phase noise, increase the accuracy of the interferometric phase, and reduce the number of interferogram residues as an aid to phase unwrapping. In this case, residues are points in the interferogram where the sum of the phase differences between pixels around a closed path is not 0.0, which indicates a jump in phase.
 
 #### Masking
-Another step before unwrapping is to create a validity mask to guide the phase unwrapping process. This mask is generated from the coherence and amplitude (backscatter intensity) values. 
+Another step before unwrapping is to create a validity mask to guide the phase unwrapping process. This mask is generated by applying thresholds to the coherence and amplitude (backscatter intensity) values for an image pair.
 
-Coherence is estimated from the normalized interferogram and the co-registered intensity images using an MLI estimator with rectangular weighting with a 5x5 moving window. The pixel values in this file range from 0.0 (total decorrelation) to 1.0 (perfectly coherent). Any input pixel with a coherence value less than 0.1 or an intensity value less than 0.2 is given a validity mask value of zero and not used during unwrapping.
+Coherence is estimated from the normalized interferogram and the co-registered intensity images using an MLI estimator with rectangular weighting with a 5x5 moving window. The pixel values in this file range from 0.0 (total decorrelation) to 1.0 (perfectly coherent). Any input pixel with a coherence value less than 0.1 is given a validity mask value of zero and not used during unwrapping.
```

Comment:
@talogan Can you verify that the intensity is considered when estimating coherence? Does this statement need to be adjusted?

Reply:
You should probably change "amplitude" to "power" to stay consistent.